results-tabulation-api/api/TallySheetVersionApi/__init__.py

```diff
@@ -165,6 +166,14 @@ def get_by_id(tallySheetId, tallySheetVersionId):
 def create(tallySheetId, body):
     request_body = RequestBody(body)
     tally_sheet = TallySheet.get_by_id(tallySheetId=tallySheetId)
+
+    # validate user inputs to prevent XSS attacks
+    if not input_is_valid(request_body.get("content")):
+        raise InvalidInputException(
```

Comment:
validating the special character and length should be applied only on "strValue" if it's not defined.

Reply:
all values are converted to str for validation. is it much better to leave it at that?